src/main/java/com/synopsys/integration/alert/provider/polaris/tasks/PolarisProjectSyncTask.java

```diff
@@ -63,6 +64,7 @@
 @Component
 public class PolarisProjectSyncTask extends ScheduledTask {
     public static final String TASK_NAME = "polaris-project-sync-task";
+    public static final String DEFAULT_CRON_EXPRESSION = BlackDuckAccumulator.DEFAULT_CRON_EXPRESSION;
```

Comment:
Can we put the constant DEFAULT_CRON_EXPRESSION in ScheduledTask or something so that we don't have providers referencing each other?